products.html

```diff
@@ -2,72 +2,130 @@
-                and
-                the
-                rest of the developing world. Despite selling hundreds if not thousands of different products,
-                most of
-                these micro-retailers do not track their sales or expenses because they cannot afford a cash
-                register
-                or other point-of-sales tool. The most organized shopkeepers may record some transactions in a
-                notebook and then laboriously transfer this information to a spreadsheet but they generally
-                still
-                do
-                not have a clear idea about how to use this data to improve profits.
-            </p>
-
-            <p>
-                The end result is that most of these businesses do not know their break-even sales, cannot
-                analyze
-                an
-                investment opportunity, nor optimize their purchasing decisions. All of these limitations
-                combine
-                to
-                keep micro-retailers operating in a sub-optimal manner, reducing profits, limiting growth and
-                making
-                life more difficult for their suppliers.
-            </p>
-
-            <h3 class="font-weight-light">The Solution</h3>
-            <p>
-
-                Our first product is a <a href="http://www.tiendate.mx/" target="_blank">point-of-sale
-                software
-                application </a>that allows micro-retailers to record all store expenses and revenues directly
-                on
-                a
-                mobile phone or tablet; an external bar code reader connected to the phone via a wireless
-                connection
-                allows them to record transactions at the product level. In exchange micro-retailers gain
-                access
-                to
-                financial reporting, personalized recommendations and additional value-added services.
-            </p>
-
-            <p>
-                All of our software applications run on Android smart phones or tablets that have touch
-                screens
-                and
-                synchronize wirelessly with our Web servers.
-            </p>
 
-        </div>
-        <div class="col-12 col-md-4 text-center">
-            <img src="/assets/img/product_phone.jpg" class="card-img" alt="Tiendatek Smartphone"/>
-            <small class="text-muted">Tiendatek Smartphone</small>
+<div class="row">
+
+    <div class="col">
+
+        <div class="card">
+            <img class="card-img-top" src="/assets/img/product_bg_tiendatek.jpg" alt="tiendatek">
+            <div class="card-body">
+
+                <a name="tiendatek"><h3>Tiendatek</h3></a>
+
+                <h4 class="font-weight-light">The problem</h4>
+                <p>
+                    Our first set of customers are the "mom-and-pop" shops that dot many corners in Latin
```

Comment:
Sé que sería un coñazo e igual no tiene fácil solución, ¿pero se pueden eliminar los saltos de linea (artificiales) en todos los textos?

En vez de:

```
America
and
the
```

`America and the`

Reply:
Con una expresión regular y Sublime yo lo he hecho otras veces. @francho , avisa si quieres ayuda con eso.

Reply:
Apañado con una expresión regular y PyCharm ;)